inst/stan/linear_single_ind.stan

```diff
@@ -1,8 +1,8 @@
 //Growth function
 functions{
   //Growth function for use with Runge-Kutta method
-  //pars = (growth_par, max_size)
-  real DE(real y, array[] real pars){ //change number of pars
+  //pars = (beta_0, beta_1, y_bar)
```

Comment:
this branch adds a vignette, but there's also some change here to R functions and Stan files. are these intended?